### ISSUE TITLE
faster rcnn demo.py all 300 object proposals without output results

### ISSUE THREAD

**Issue by Anhaoxu** [rbgirshick/py-faster-rcnn#512]
@rbgirshick @Dectinc @wangdelp 
Hi, I use windows10 cpu to run fsater rcnn demo.py with zf models,but no pictures to display,I check it many ways,I think it nothing wrong about python,because it can run very well in fast rcnn.I can understand the problem is that it didn't deal with the demo pictures,even though it runs no warnings and wrongs,but I can't understand why? and how I can solve this problem.They are all 300 object proposals and no pictures to display.Thank you for help me!

 I0305 12:00:55.676915 13204 net.cpp:771] Ignoring source layer pool5_spm6
 I0305 12:00:55.677916 13204 net.cpp:771] Ignoring source layer pool5_spm6_flatten
 I0305 12:00:55.682919 13204 net.cpp:774] Copying source layer fc6
 I0305 12:00:55.733954 13204 net.cpp:774] Copying source layer relu6
 I0305 12:00:55.734513 13204 net.cpp:774] Copying source layer drop6
 I0305 12:00:55.734513 13204 net.cpp:774] Copying source layer fc7
 I0305 12:00:55.758543 13204 net.cpp:774] Copying source layer relu7
 I0305 12:00:55.759537 13204 net.cpp:774] Copying source layer drop7
 I0305 12:00:55.759537 13204 net.cpp:771] Ignoring source layer fc8
 I0305 12:00:55.760531 13204 net.cpp:771] Ignoring source layer prob

 Loaded network C:\Users\shihaohou\Desktop\py-faster-rcnn-  master\data\faster_rcnn_models\ZF_faster_rcnn_final.caffemodel
 <caffe._caffe.Net object at 0x000000000EC21F48>
 <caffe._caffe.Net object at 0x000000000EC21F48>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Demo for data/demo/000456.jpg
 Detection took 3.657s for 300 object proposals
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Demo for data/demo/000542.jpg
 Detection took 3.526s for 300 object proposals
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Demo for data/demo/001150.jpg
 Detection took 3.591s for 300 object proposals
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Demo for data/demo/001763.jpg
 Detection took 3.506s for 300 object proposals
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Demo for data/demo/004545.jpg
 Detection took 3.529s for 300 object proposals

**Comment by youssef-osaro** [comment]
I'm having a similar problem with the demo. No results are being displayed because the network is basically detecting nothing, all the output scores are almost zero except for the background class with score almost 1.

**Comment by Anhaoxu** [comment]
@youssef-osaro yeah, but now I meet another problem,there are too many bbox in the displayed pictures, how can I get one over these boxes?

**Comment by boyflytobeman** [comment]
@Anhaoxu ,i met the same problem,have you solved the problem?

**Comment by aykutonol** [comment]
I have the same problem too...

**Comment by yayo13** [comment]
@Anhaoxu @youssef-osaro @boyflytobeman I've met the same issue, but when i changed to CPU-MODE, it  works! i don't know why...

**Comment by sulth** [comment]
I got a good model output but couldnt display the images using plt.show().Do anyone know how to display it?
